src/drive/mobile/lib/intents.js

```diff
@@ -58,6 +58,18 @@ const getFiles = contentFiles =>
     })
   )
 
+const base64toBlob = (dataURI, type) => {
+  const byteString = atob(dataURI)
+  const ab = new ArrayBuffer(byteString.length)
+  const ia = new Uint8Array(ab)
+
+  for (let i = 0; i < byteString.length; i++) {
+    ia[i] = byteString.charCodeAt(i)
+  }
+
+  return new Blob([ab], { type })
+}
```

Comment:
The cordova plugin sends image files as base64 encoded strings, so we need to turn them into blobs.